LICENSE.txt

```diff
@@ -1,27 +1,14 @@
+BSD 3-clause license
```

Comment:
This is the license file for the staged-recipes repository and should not be changed by this PR.

Reply:
I change back to the original LICENSE